frontend/src/lib/utils.tsx

```diff
@@ -619,19 +619,45 @@ export function isEmail(string: string): boolean {
     return !!string.match?.(regexp)
 }
 
-export function eventToName(event: Pick<EventType, 'elements' | 'event' | 'properties'>): string {
+export function eventToName(
```

Comment:
Nit: maybe `eventToDescription` is more appropriate? Feels like the event name is `$pageview` etc. Same for `autoCaptureEventToName`